Astral Pathfinder/Models/Game Object/ColliderComponent.hpp

```diff
@@ -9,32 +9,56 @@
 #ifndef ColliderComponent_hpp
 #define ColliderComponent_hpp
 
+// MARK: Libraries and Frameworks
 #include "SDL2/SDL.h"
 #include <vector>
+#include <experimental/optional>
```

Comment:
da fuq is this

Reply:
Optionals allow for you to have a variable that may or may not be valid. A better way to handle null values instead of using a nullptr or NULL.